integration-tests/testconfig/testconfig.go

```diff
@@ -157,13 +156,19 @@ func (c *TestConfig) Save() (string, error) {
 }
 
 // MustCopy Returns a deep copy of the Test Config or panics on error
-func (c TestConfig) MustCopy() any {
-	return deepcopy.MustAnything(c).(TestConfig)
-}
+func (c *TestConfig) MustCopy() TestConfig {
```

Comment:
Does it make sense to have it as util method in CTF? which will take any interface return the copy?

Reply:
it does :-)